sdk/v2/system/api_client.go

```diff
@@ -16,8 +16,11 @@ type PingResponse struct {
 
 // APIClient is the client for system checks involving the Brigade API.
 type APIClient interface {
-	// Ping pings the API Server
+	// Ping sends a GET request to the API Server's versioned ping endpoint
 	Ping(ctx context.Context) (PingResponse, error)
+
+	// PingRaw sends a GET request to the API Server's unversioned ping endpoint
+	PingRaw(ctx context.Context) (string, error)
```

Comment:
Nit: I'm not sure how I feel about "PingRaw" because (unless one as has read the code), the name doesn't unambiguously denote its function. It returns a _version_ string, but nothing about the name hints that that is what you'll get when you invoke this.

imho, if we named this `UnversionedPing` its function would be more obvious.

What would you think about still returning JSON instead of just a string? That leaves the possibility of including additional information in the response someday if we deem it necessary. Already, I can imagine that there are _two_ useful bit of info to returns-- server version and server _API_ version.

Then there's the "raw" aspect of it. If we don't assume a particular API version, we don't know what sort of type to unmarshal the response into. Maybe we can just return a slice of bytes?

Something like this?

```
UnversionedPing(ctx context.Context) ([]byte, error)
```

Reply:
> If we don't assume a particular API version, we don't know what sort of type to unmarshal the response into.

Yea, that's what I was thinking, hence returning a string (and not structured JSON).  But I'm down to switch to `[]byte` for sure.

`UnversionedPing` sgtm!